pkg/cli/admin/release/extract_tools.go

```diff
@@ -492,17 +492,16 @@ func (o *ExtractOptions) extractCommand(command string) error {
 
 		// copy the input to disk
 		if target.InjectReleaseImage {
```

Comment:
Why are we using release image and release image name under the same flag?  They are unrelated - oc and Openshift-install should be using this flag for replace, rather than checking on binary name.

Reply:
good point, thanks! _reworking_

Reply:
pushed an update, added `InjectReleaseVersion` bool